src/components/Tag.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="tag">
+    <a href="#" class="btn-icon"><p class="tag-name"># {{ tagName }}</p></a>
```

Comment:
下記でもいい気がしたけど何か理由ってあったりする？
```suggestion
    <a href="#" class="btn-icon"><p class="tag-name">#{{ tagName }}</p></a>
```